### PR TITLE
add ibc and update logo

### PR DESCRIPTION
**Pull request by ShenFri**

### CODE REVIEW

File: _IBC/uptick-cosmos.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "uptick",
```
Comment (JeremyParish69) [inline _IBC/uptick-cosmos.json:4]: for both ibc files, they must be ordered alphabetically, so they should be named cosmoshub-uptick.json, with chain 1 being cosmoshub, and chain 2 being uptick

File: _IBC/uptick-iris.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "uptick",
+    "client_id": "07-tendermint-2",
+    "connection_id": "connection-0"
+  },
+  "chain_2": {
+    "chain_name": "iris",
```
Comment (JeremyParish69) [inline _IBC/uptick-iris.json:9]: same here, irisnet should be chain 1, and uptick as chain 2 (and file renamed accordingly)

Comment (ShenFri) [reply _IBC/uptick-iris.json]: ok

**Comment by JeremyParish69** [comment]
Oh, and the 'version' property is required in both files (e.g., `"version": "ics20-1"`)

**Pull request merged**